pytorch_vision_efficientnet.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: EfficientNet
+summary: Memory and Performance Efficient Networks with 8 configurations.
+category: researchers
+image: efficientnet1.png
+author: Pytorch Team
+tags: [vision, scriptable]
+github-link: https://github.com/pytorch/vision/blob/main/torchvision/models/efficientnet.py
+github-id: pytorch/vision
+featured_image_1: efficientnet1.png
+featured_image_2: no-image
+accelerator: cuda-optional
+order: 10
+---
+
+```python
+import torch
+model = torch.hub.load('pytorch/vision:v0.11.0', 'efficientnet_b0', pretrained=True)
```

Comment:
I think that it would be wrong to mention `pytorch/vision:v0.10.0` as the model was not added till then.
It will be available in the coming 0.11 release.

Should we merge efficientnet now or after the 0.11 release?
I'm happy either ways, we can keep a  separate PR open and merge after the release.

Reply:
Yeah that's a good point... Good call. Do you want to remove this from the PR and send the addition on a separate? We could leave it unmerged until the release.

Reply:
Cool! I will remove the efficientnet files from current PR and keep them on separate one.
As you suggested, we can merge it after release :+1: